### PR TITLE
feat: Card component add

### PR DESCRIPTION
**Pull request by ajitpsakri**
## Features

1. Basic card
2. Social media card 
3. Dismissable card
4. Text overlay card
5. E-commerce card

## Screen-shot
image

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+}
+.card-text-overlay-header i {
+    font-size: 2rem;
+    margin-right: 2rem;
+}
+
+.card-text-overlay-footer .card-icons{
+    position: absolute;
+    bottom: 0;
+    color: white;
```
Comment (RakshaPawar108) [inline components/card/card.css]: You can create variables for colors instead of directly writing names or hex codes

**Review by AtulPant2704 — APPROVED** [review]

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+    font-size: 6rem;
+    color: red;
+    /* background-color: red; */
+}
+.card-ecom-header-offer-icon p{
+    position: absolute;
+    top: 0.5rem;
+    left: 1rem;
+    font-size: 1.5rem;
+    /* margin: 0.5rem; */
```
Comment (RakshaPawar108) [inline components/card/card.css]: Remove the commented code if not required to make codebase look cleaner

**Review by RakshaPawar108 — COMMENTED** [review]
Apart from the suggested changes, the rest of the code looks good to merge

**Review by RakshaPawar108 — APPROVED** [review]

File: components/card/card.css (hunk 1 above)
Comment (ajitpsakri) [reply components/card/card.css]: Ok got it. Thanks for the review

**Pull request merged**